keps/sig-node/2400-node-swap/README.md

```diff
@@ -487,14 +572,14 @@ Test grid tabs enabled:
 
 No new e2e tests introduced.
 
-For alpha2 [Current stage]:
+For alpha2:
 
 - Add e2e tests that exercise all available swap configurations via the CRI.
 - Verify MemoryPressure behavior with swap enabled and document any changes
   for configuring eviction.
 - Verify new system-reserved settings for swap memory.
 
-For beta [Future]:
+For beta 1:
 
 - Add e2e tests that verify pod-level control of swap utilization.
 - Add e2e tests that verify swap performance with pods using a tmpfs.
```

Comment:
I would be happy for some clarifications regarding the test plan for Beta1.

> Add e2e tests that verify pod-level control of swap utilization.

This sounds a bit outdated. Can we change it to something like `Add e2e tests that verify that the right amount of swap limitation is automatically configured for burstable pods`?

> Add e2e tests that verify swap performance with pods using a tmpfs.

@SergeyKanzhelev would you mind clarifying that?
Which performance test do we want to do here? IOW, what do we want to verify?
Also, what does "pods using a tmpfs" means in this context?

Thanks!

Reply:
@SergeyKanzhelev sorry but I was also not able to gather any context around this line, `Add e2e tests that verify swap performance with pods using a tmpfs.` I see that it was added when swap was introduced as alpha.